micro_sam/training/medsam_trainer.py

```diff
@@ -0,0 +1,20 @@
+from . import SamTrainer
+
+
+class MedSAMTrainer(SamTrainer):
+    """Trainer class for replicating the trainer of MedSAM (https://arxiv.org/abs/2304.12306)
+    """
+    def __init__(
+        self,
+        **kwargs
+    ):
+        super().__init__(**kwargs)
```

Comment:
We should set `n_sub_iterations` to 1 here. It doesn't make sense to require it as a kwarg.

Reply:
Done. Fixed the `n_sub_iteration` and `mask_prob` values in the trainer. Thanks for the mention!